src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
+        Color randomColor = ColorSupplier.getRandomColor();
```

Comment:
color supplier also should be class field as it does not should have static keyword